file.hh

```diff
@@ -1,19 +1,18 @@
 /* This file is (c) 2008-2012 Konstantin Isakov <ikm@goldendict.org>
  * Part of GoldenDict. Licensed under GPLv3 or later, see the LICENSE file */
 
-#ifndef __FILE_HH_INCLUDED__
-#define __FILE_HH_INCLUDED__
+#ifndef GOLDENDICT_FILE_HH
+#define GOLDENDICT_FILE_HH
```

Comment:
Why change this?

Reply:
C++ claim all `__` as reversed names, some IDEs/Tools keep showing it as warning and it is annoying :sweat_smile: 

https://en.cppreference.com/w/cpp/keyword

![image](https://user-images.githubusercontent.com/20123683/232041026-3b38a3bc-1dc1-4e1c-83b9-c4837a898773.png)

Reply:
In fact, I think we should just use `#pragma once`

Transmission fully migrated from C to C++ recently, and their entire codebase just use `#pragma once`

https://github.com/transmission/transmission/blob/6b0e49bbb296f1c84785275b8a8f18b4210180af/qt/AboutDialog.h#L6

Reply:
Is `#pragma once   ` not a standard yet?

Reply:
It is a quasi / de facto standard, since all compilers implemented it :sweat_smile:

Reply:
![image](https://user-images.githubusercontent.com/20123683/232048178-cb0dcbce-a5e6-4fc9-9f2b-190b4f7e4f46.png)

gcc/msvc/clang all have special optimizations about `#progma once` to speedup compilation

https://learn.microsoft.com/en-us/cpp/preprocessor/once?view=msvc-170

Reply:
At least we can ensure all new incoming files use this